lib/puppetserver/ca/action/list.rb

```diff
@@ -97,7 +97,7 @@ def output_certs(certs)
 
           certs.each do |cert|
             @logger.inform "    #{cert["name"]}".ljust(padded + 6) + " (SHA256) " + " #{cert["fingerprints"]["SHA256"]}" +
-                               (cert["dns_alt_names"].empty? ? "" : "\talt names: #{cert["dns_alt_names"]}")
+                               (cert["subject_alt_names"].empty? ? "" : "\talt names: #{cert["subject_alt_names"]}")
```

Comment:
We should see if subject_alt_names exists and if so use it, otherwise use dns_alt_names.

Reply:
We can do that later